### PR TITLE
[Enterprise-3.11] Added a new section to sysctls to mention how to handle unsafe sysctls

### LINKED ISSUE #15234: [enterprise-3.11] Edit suggested in file admin_guide/sysctls.adoc
### Which section(s) is the issue in?

Needs a new section

### What needs fixing?

In the [release notes](https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html#ocp-311-notable-technical-changes) it has a section detailing how Sysctl and SCC interact now.

> ### Forbidden and unsafe sysctls options
> Security Context Constraints have two new options to control which sysctl options can be defined in a pod spec:
>
> - The forbiddenSysctls option excludes specific sysctls.
> - The allowedUnsafeSysctls option controls specific needs such as high performance or real-time application tuning.
>
> All safe sysctls are enabled by default; all unsafe sysctls are disabled by default and must be manually allowed by the cluster administrator.

But there is no mention of this on the Sysctls page. Took me quite a while (and a hand from our TAM) to figure it out. I think the info from the Release Notes needs to also be on this page, possibly expanded a bit to fit with the other examples on the page.

### PR DESCRIPTION
**Pull request by EricPonvelle**
Fixes #15234. This burn down ticket requested information in the documentation about the forbidden and unsafe sysctls options. For this update, I added the two options to the 3.11 documentation. A separate ticket will be needed for 4.5+ documentation.

Link to section: https://deploy-preview-30944--osdocs.netlify.app/openshift-enterprise/latest/admin_guide/sysctls.html#enabling-unsafe-sysctls

I add the statement and the two bullets on the two options for Unsafe sysctls.

### CODE REVIEW

**Comment by EricPonvelle** [comment]
Hi, @xiaojiey!

Can you review the items I changed in the 3.11 documentation. I'll make a second ticket for the updates to 4.5+ next if this is still valid. Did I fully cover what these parameters do?

File: admin_guide/sysctls.adoc [hunk 1]
```diff
@@ -118,6 +118,11 @@ situations such as high-performance or real-time application tuning.
 If you want to use unsafe sysctls, cluster administrators must enable them
 individually on nodes. They can enable only namespaced sysctls.
 
+There are two methods to control which sysctl options can be defined:
+
+- `forbiddenSysctls` option excludes specific sysctls.
+- `allowedUnsafeSysctls` option controls specific needs such as high performance or real-time application tuning.
```
Comment (bobfuru) [inline admin_guide/sysctls.adoc:124]: There is IBM Style Guide (ISG) guidance to not start a sentence with a lowercase word when possible. I suggest adding "The" at the beginning of each option.

**Review by bobfuru — COMMENTED** [review]
A couple of minor suggestions, otherwise LGTM!

File: admin_guide/sysctls.adoc (hunk 1 above)
Comment (bobfuru) [reply admin_guide/sysctls.adoc:124]: Also, I noticed that "high performance" is hyphenated previously in line 116. I prefer your version, as I think "high performance" is a noun and not an adjective describing app tuning (i.e., "high-performance application tuning"). So maybe just drop the hyphen in the previous instance?

Comment (EricPonvelle) [reply admin_guide/sysctls.adoc]: Great catches. I applied and pushed those changes. I was a little hesitant on the hyphen, but reading in context, it definitely is a noun.

@sunilcio can you review these changes and let me know if I covered everything accurately? I need to make another ticket, I believe, to update 4.5+ with these changes if they are validated. Thanks!

**Comment by sunilcio** [comment]
@EricPonvelle thanks for edit, LGTM.

**Comment by barleyer** [comment]
> Hi, @xiaojiey!
> 
> Can you review the items I changed in the 3.11 documentation. I'll make a second ticket for the updates to 4.5+ next if this is still valid. Did I fully cover what these parameters do?

I think the description of ` You can further control which sysctls can be set in pods by specifying lists of sysctls or sysctl patterns in the forbiddenSysctls and/or allowedUnsafeSysctls fields of the Security Context Constraints.` is clearer than `There are two methods to control which sysctl options can be defined:` **forbiddenSysctls** and **allowedUnsafeSysctls** are two options/fields  of SCC,not two methods.

File: admin_guide/sysctls.adoc [hunk 2]
```diff
@@ -113,11 +113,16 @@ launch.
 == Enabling unsafe sysctls
 
 The cluster administrator can allow certain unsafe sysctls for very special
-situations such as high-performance or real-time application tuning.
+situations such as high performance or real-time application tuning.
 
 If you want to use unsafe sysctls, cluster administrators must enable them
 individually on nodes. They can enable only namespaced sysctls.
 
+You can further control which sysctls can be set in pods by specifying lists of sysctls or sysctl patterns in the `forbiddenSysctls` and/or `allowedUnsafeSysctls` fields of the Security Context Constraints.
```
Comment (bobfuru) [inline admin_guide/sysctls.adoc:121]: Per ISG, drop `/or` here:
and/or -> and

"Do not use the construction and/or, which can cause confusion and problems for translation.
Instead, clarify the options by using and or or, or by stating the choices, such as
option A, option B, or both."

Comment (EricPonvelle) [reply admin_guide/sysctls.adoc]: Hey @bobfuru. I appreciate the guidance. I updated the text.

**Comment by bobfuru** [comment]
Updates LGTM

**Pull request merged**